llvm/test/Transforms/MemCpyOpt/memcpy_invalid_modify.ll

```diff
@@ -0,0 +1,39 @@
+; RUN: opt -S -passes=memcpyopt < %s -verify-memoryssa | FileCheck %s
```

Comment:
Use update_test_checks.py.

Reply:
Thanks for reply. I used it in my new modification.

Reply:
It doesn't look that way. Please use update_test_checks.py to generate the CHECK lines and then **do not manually edit them**.